engine/crates/parser-sdl/src/rules/visitor/context.rs

```diff
@@ -184,6 +185,7 @@ impl<'a> VisitorContext<'a> {
         }
 
         registry.remove_unused_types();
+        registry.remove_empty_types();
```

Comment:
I wonder about the position of this call as well actually.  This is going to be called whenever _any_ registry is finalised, so if a user gives us `type Blah` without any fields we'll just silently remove `Blah` and never complain.

I wonder if this call would be better off called against `src_registry` here: https://github.com/grafbase/grafbase/blob/main/engine/crates/parser-sdl/src/connector_parsers.rs#L63 so it only acts on connector generated registries and not on anything a user gives us....

Reply:
Good point, that indeed sounds like a better solution.

Reply:
done